src/util/seeds.ts

```diff
@@ -213,7 +213,12 @@ export class ScopedSeed {
     return depth >= this.maxDepth;
   }
 
-  isIncluded(url: string, depth: number, extraHops = 0, logDetails = {}) {
+  isIncluded(
+    url: string,
+    depth: number,
+    extraHops = 0,
+    logDetails = {},
+  ): { url: string; isOOS: boolean } | false {
```

Comment:
This return type works but I wonder if we want to try to be more consistent?

Reply:
well, typescript has helped make this more consistent! I feel like this works for now and is type-safe, so maybe lets leave in as is for now.